### DISCUSSION TITLE
Do you have a `.deb` release? I can help you with one

### DISCUSSION THREAD

**Discussion by John15321** [cargo-generate/cargo-generate#652] [Ideas]
Do you have a `.deb` release? I can help you with one.
If I am mistaken, please correct me, but I cannot find a Debian package release of your project. I can see in your book that you have instructions for installing on Arch but not for other distros. I don't think every distro is worth it ofc but I think having one for Debian-based distros would be nice. Im currently doing the same for `cargo-make`

**Comment by sassman** [comment]
@John15321 we had one in the past, but then It got a bit hairy with maintaining it. I created a tiny deb builder action https://github.com/sassman/rust-deb-builder but it's always a bit behind and needs more maintenance.
